pinot-controller/src/main/resources/app/components/Homepage/Operations/RebalanceServer/RebalanceServerResponses/RebalanceServerResponseLabelValue.tsx

```diff
@@ -25,7 +25,7 @@ export const RebalanceServerResponseLabelValue = (
     return (
         <Box>
             <Typography color='textSecondary' variant='caption'>{label}</Typography>
-            <Typography style={{ fontWeight: 600 }} variant='body2'>{value}</Typography>
+            <Typography style={{ fontWeight: 600 }} variant='body2'>{value.split("\n").join("<br />")}</Typography>
```

Comment:
no this only handles \n, what about other whitespaces chars like \t, enter etc...
ref - https://www.w3schools.com/cssref/pr_text_white-space.php

Reply:
Done!